frontstage_api/resources/sign_in/sign_in.py

```diff
@@ -19,8 +19,8 @@ class SignIn(Resource):
     def post():
         logger.info('Attempting to retrieved sign-in details')
         message_json = request.get_json(force=True)
-        username = message_json['username']
-        password = message_json['password']
+        username = message_json.get('username')
```

Comment:
I kind of prefer the square brackets here to `.get`. If we don't have a username or password this is going to fail anyway and this will at least spit out a 400 with the square ones.
We'll add marshmallow validation for these endpoints anyway at some point to handle that kind of thing at some point

Reply:
I can't see where the KeyValue error will get caught? With the square ones it looks like it would crash instead of give a 400. but i'm happy to change it back if we'll be adding validation in later.